mautic/mautic
# Fix social monitoring command for twitter

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #4100
| BC breaks? | 
| Deprecations? | 


[//]: # ( Required: )
#### Description:

When you launch this command : mautic:social:monitoring, it fail with an error : 

[Symfony\Component\Debug\Exception\ContextErrorException]
Catchable Fatal Error: Argument 2 passed to Mautic\PluginBundle\Helper\oAuthHelper::__construct() must be an instance of Symfony\Component\HttpFoundation\Request, null given, called in /home/contactsiwantasp/public_html/app/bundles/PluginBundle/Integration/AbstractIntegration.php on line 1119 and defined

⚠️ This fix is not a real solution I think, but it fixes. ⚠️ 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Config the twitter plugin
2. Create a social monitoring
3. Launch `mautic:social:monitoring`
4. See the error, is the same in the description of this PR.

#### Steps to test this PR:
1. Apply this PR.
2. Config the twitter plugin
3. Create a social monitoring
4. Launch `mautic:social:monitoring`
5. See the lead create or update.
